mitiq/shadows/test/test_quantum_processing.py

```diff
@@ -159,17 +159,15 @@ def test_random_pauli_measurement_output_dimensions(
     shadow_outcomes, pauli_strings = random_pauli_measurement(
         circuit, n_total_measurements, executor=executor
     )
-    assert shadow_outcomes.shape == (n_total_measurements, n_qubits), (
+    shadow_outcomes_shape = len(shadow_outcomes), len(shadow_outcomes[0])
+    pauli_strings_shape = len(pauli_strings), len(pauli_strings[0])
```

Comment:
TIL you can assign a variable as a tuple with surrounding the expression in parens! Cool 😎.